ofrak_core/Dockerstub

```diff
@@ -63,3 +63,13 @@ WORKDIR /tmp
 RUN wget https://bootstrap.pypa.io/pip/get-pip.py && python3.9 get-pip.py && python3.7 get-pip.py && rm get-pip.py
 RUN python3.9 -m pip install jefferson
 WORKDIR /
+
+# Install UEFIExtract
+# TODO: This is broken - missing some deps to run
```

Comment:
If you recompile so it uses the right version of GLIBC, then it should also work without the `$TARGETARCH` check.